csharp/ql/src/semmle/code/csharp/frameworks/Format.qll

```diff
@@ -218,10 +227,12 @@ class FormatCall extends MethodCall {
   }
 
   /**
+   * DEPRECATED: Use `FormatFlow` module instead.
```

Comment:
Perhaps update comment to ``DEPRECATED: Use `FormatFlow::hasFlowPath` instead.``